core/src/test/scala/unit/kafka/server/LogOffsetTest.scala

```diff
@@ -42,7 +42,7 @@ class LogOffsetTest extends ZooKeeperTestHarness {
   var logDir: File = null
   var topicLogDir: File = null
   var server: KafkaServer = null
-  var logSize: Int = 100
+  var logSize: Int = 120
```

Comment:
Yes, @apurvam created a JIRA to make these test cases more resilient to message format changes. For now, I found a value that works.